client/a8c-for-agencies/data/agencies/use-fetch-agency-from-blog.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+
+export default function useFetchAgencyFromBlog(
+	blogId: number,
+	options: { enabled?: boolean } = {}
+) {
+	const { enabled = true, ...restOptions } = options;
+
+	return useQuery( {
+		queryKey: [ 'agency-blog', blogId ],
+		queryFn: () =>
+			wpcom.req.get( {
+				apiNamespace: 'wpcom/v2',
+				path: `/agency/blog/${ blogId }`,
+			} ),
+		select: ( data ) => {
+			return {
+				id: data?.id,
+				name: data?.name,
+			};
+		},
```

Comment:
In this case, I think the select is not needed since the attributes are exactly the same.

Reply:
Thank you, Paulo. Even though the attributes are the same, I used the `select` here because the endpoint itself is returning more information than we need at the moment (id and the name).

Reply:
Ah ok, I didn't notice that. All good then. 😄